src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const nodesList = document.querySelectorAll('[data-salary]');
+
+function prepareNumber(string) {
+  return +string
+    .slice(1)
+    .split(',')
+    .join('');
+}
+
+function sortList(list) {
+  const listArray = Array.from(list);
+
+  listArray.sort((a, b) => (
+    prepareNumber(b.dataset.salary) - prepareNumber(a.dataset.salary)
+  ));
+
+  for (const node of listArray) {
```

Comment:
Consider using `forEach` instead of the `for...of` loop to iterate over the `listArray`. This will make the code more functional and easier to read